app/helpers/brexit_checker_helper.rb

```diff
@@ -91,4 +91,17 @@ def brexit_results_description(actions, criteria_keys)
       t("brexit_checker.results.description_no_answers").html_safe
     end
   end
+
+  def account_signup_jwt(criteria_keys)
+    account_jwt = BrexitChecker::AccountJwt.new(criteria_keys)
+    account_jwt.encode
+  end
+
+  def path_based_on_account_feature_flag
+    if ENV["FEATURE_FLAG_ACCOUNTS"] == "enabled"
+      transition_checker_save_results_path(c: criteria_keys)
```

Comment:
_path, or _url? does it matter for this?

Reply:
`path` is fine